cmd/dashboard/repl.js

```diff
@@ -36,6 +50,18 @@ class REPL {
       process.exit();
     });
 
+    this.replServer.defineCommand('history', {
+      help: 'Show last history',
+      action: function(_n) {
+        let n = _n || 10;
+        console.log(this.history
+                      .slice(0, n)
+                      .filter(line => line.trim())
+                      .join('\n'));
+        this.displayPrompt();
+      }
+    });
```

Comment:
You should use the plugin reference (https://embark.status.im/docs/plugin_reference.html#embark-registerConsoleCommand-callback-options)  so that it can be available both in the dashboard and in the console. Also, put the event in the command_history module.

Reply:
Hi @jrainville thank you for the review. I will start working on this today.